Install fails (npm install barrelsby).

Installing this module via NPM fails. 

```
MacBook-Pro:plo-canvas user$ npm install barrelsby
npm ERR! Darwin 16.5.0
npm ERR! argv "/usr/local/Cellar/node/7.10.0/bin/node" "/usr/local/bin/npm" "install" "barrelsby"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0
npm ERR! path /Users/user/Developer/project/node_modules/barrelsby/bin/index.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/user/Developer/project/node_modules/barrelsby/bin/index.js'
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/user/Developer/project/node_modules/barrelsby/bin/index.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
```

I don't know for sure, but from what I've gathered, NPM is using your .gitignore file as the .npmignore file. You should use a separate .npmignore file.

--- a/package.json
+++ b/package.json
@@ -50,9 +50,6 @@
     "tslint": "^5.0.0",
     "typescript": "^2.1.6"
   },
-  "files": [
-    "./bin/index.js"
-  ],
   "bin": {
     "barrelsby": "bin/index.js"
   },

